scripts/weaviate-test-helpers/pdf_elements.json

```
@@ -0,0 +1,413 @@
+                },
+                {
+                    "dataType": [
+                        "date"
+                    ],
+                    "indexFilterable": true,
+                    "indexSearchable": false,
+                    "name": "last_modified"
+                },
+                {
+                    "dataType": [
+                        "text"
+                    ],
+                    "indexFilterable": true,
+                    "indexSearchable": true,
+                    "name": "file_directory",
+                    "tokenization": "word"
+                },
+                {
+                    "dataType": [
+                        "text"
+                    ],
+                    "indexFilterable": true,
+                    "indexSearchable": true,
+                    "name": "filename",
+                    "tokenization": "word"
+                },
+                {
+                    "dataType": [
+                        "object"
+                    ],
+                    "indexFilterable": true,
+                    "indexSearchable": false,
+                    "name": "data_source",
+                    "nestedProperties": [
+                        {
+                            "dataType": [
+                                "text"
+                            ],
+                            "indexFilterable": true,
+                            "indexSearchable": true,
+                            "name": "url",
+                            "tokenization": "word"
+                        },
+                        {
+                            "dataType": [
+                                "text"
+                            ],
+                            "indexFilterable": true,
+                            "indexSearchable": true,
+                            "name": "version",
+                            "tokenization": "word"
+                        },
+                        {
+                            "dataType": [
+                                "date"
+                            ],
+                            "indexFilterable": true,
+                            "indexSearchable": false,
+                            "name": "date_created"
+                        },
+                        {
+                            "dataType": [
+                                "date"
+                            ],
+                            "indexFilterable": true,
+                            "indexSearchable": false,
+                            "name": "date_modified"
+                        },
+                        {
+                            "dataType": [
+                                "date"
+                            ],
+                            "indexFilterable": true,
+                            "indexSearchable": false,
+                            "name": "date_processed"
+                        },
+                        {
```

Might want to support permissions data, even if as a plain string

```suggestion
                        {
                            "dataType": [
                                "text"
                            ],
                            "indexFilterable": true,
                            "indexSearchable": true,
                            "name": "permissions_data",
                            "tokenization": "word"
                        },

                        {
```